Realm/RLMArray.h

```diff
@@ -264,7 +263,7 @@ NS_ASSUME_NONNULL_BEGIN
  Returns a sorted `RLMResults` from the array.
 
  @param property    The property name to sort by.
- @param ascending   The direction to sort by.
+ @param ascending   The direction to sort.
```

Comment:
I don't think it's correct to say that you "sort by" a direction. I'd say that you sort by a property _in_ a direction.